### DISCUSSION TITLE
Prisma - How to Make Variable Product Schema?

### DISCUSSION THREAD

**Discussion by AminAppDevs** [prisma/prisma#13438] [Q&A]
I want to make variable product schema in prisma, like woocommerce variable product.
[this is my model, how can i make it for simple and variable product.](https://i.stack.imgur.com/YsdqA.png)

**Comment by nurul3101** [comment]
Hey @AminAppDevs 👋 
Can you please elaborate, what do you mean by variable product schema?

**Reply by vladi-stevanovic** [reply]
👋 Hello @AminAppDevs  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
